### ISSUE TITLE
asset_tracker_v2 (v1.7.1) disconnects on FOTA job request with v1.6.x

### ISSUE THREAD

**Issue by coderbyheart** [NordicSemiconductor/asset-tracker-cloud-aws-js#633]
```
[00:00:14.219,116] <dbg> aws_iot.aws_iot_cloud_poll: AWS IoT broker connection request sent
[00:00:14.419,433] <dbg> aws_jobs.reg_topic: Subscribe: $aws/things/352656106120200/jobs/notify-next
[00:00:14.430,023] <dbg> aws_jobs.reg_topic: Subscribe: $aws/things/352656106120200/jobs/$next/get/#
[00:00:14.440,399] <dbg> aws_iot.mqtt_evt_handler: MQTT client connected!
[00:00:14.447,540] <dbg> aws_iot_integration.aws_iot_event_handler: AWS_IOT_EVT_CONNECTED
[00:00:14.456,176] <dbg> aws_iot.topic_subscribe: Subscribing to application topic: $aws/things/352656106120200/shadow/get/accepted/desired/cfg
[00:00:14.469,512] <dbg> aws_iot.topic_subscribe: Subscribing to application topic: 352656106120200/agps
[00:00:14.479,431] <dbg> aws_iot.topic_subscribe: Subscribing to application topic: 352656106120200/pgps
[00:00:14.490,173] <dbg> aws_iot.topic_subscribe: Subscribing to AWS shadow topic: $aws/things/352656106120200/shadow/get/rejected
[00:00:14.502,319] <dbg> aws_iot.topic_subscribe: Subscribing to AWS shadow topic: $aws/things/352656106120200/shadow/update/delta
[00:00:14.583,343] <dbg> aws_fota.on_suback_evt: Subscribed to notify-next topic
[00:00:14.591,186] <dbg> aws_jobs.publish: Publish topic: $aws/things/352656106120200/jobs/$next/get
[00:00:14.600,677] <dbg> aws_jobs.publish: Publish payload {"clientToken": ""}
[00:00:14.609,252] <dbg> aws_iot.mqtt_evt_handler: MQTT_EVT_SUBACK: id = 2112 result = 0
[00:00:14.850,982] <dbg> aws_iot.mqtt_evt_handler: MQTT_EVT_DISCONNECT: result = -128
[00:00:14.859,222] <dbg> aws_iot_integration.aws_iot_event_handler: AWS_IOT_EVT_DISCONNECTED
[00:00:14.868,103] <dbg> cloud_module.cloud_wrap_event_handler: CLOUD_WRAP_EVT_DISCONNECTED
[00:00:14.876,953] <inf> event_manager: CLOUD_EVT_DISCONNECTED
```

**Comment by coderbyheart** [comment]
Reported by Tech Support. Unfortunately [the docs](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.7.1/nrf/applications/asset_tracker_v2/doc/asset_tracker_v2_description.html#overview) link to 1.6.x which *might* be incompatible with the latest NCS version, or there is a bug in asset_tracker_v2 from 1.7.1.

**Comment by chshzh** [comment]
Test NCS1.6.1 version of asset_tracker_v2, it works without this issue.

**Comment by coderbyheart** [comment]
Fixed via 08cd2243980eceb5b722cb375a52b99a94c954ab

We introduced nRF Connect for Cloud A-GPS and P-GPS support in NCS 1.7, and the device now subscribes to the topics `<deviceId>/agps` and `<deviceId>/pgps`, which the existing IoT policy did not allow. This caused the disconnect.

Please update your installation by pulling this change and deploying it:

```bash
git pull
npx tsc
npx cdk deploy ${STACK_NAME}
```

A new policy will be created, and the old one deleted.

Afterwards, re-attach the `${STACK_NAME}-thingPolicy-...` to the `${STACK_NAME}` thing group in the AWS IoT console.
